msalsas/gdpr-consent-banner-bundle/1.0/config/packages/msalsas_gdpr_consent_banner.php

```diff
@@ -5,8 +5,5 @@
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
-    $containerConfigurator->extension('msalsas_gdpr_consent_banner', [
-        'fade_time' => 2,
-        'time_to_expire' => '30 days',
-    ]);
+    $containerConfigurator->extension('msalsas_gdpr_consent_banner', ['fade_time' => 2, 'time_to_expire' => '30 days']);
```

Comment:
need to check if we can fix this unexpected code style issue in: https://github.com/schranz-php-recipes/symfony-recipes-yaml-to-php-converter/pull/10